### PR TITLE
Adjustments to default.hbs, the locales folder and inclusion of the pt-BR language

### PR DESCRIPTION
**Pull request by matribeiro15**
Hello, this is my contribution at the moment.
I noticed that the en.json file had some missing items. I inserted what I thought was missing.
The data in de.json, I only entered the data in English, as I have no knowledge of the language.
I inserted translation assistant in defalt.hbs.
And finally, I inserted the pt-BR.json.

I hope I have helped.

### CODE REVIEW

File: content/themes/candidus/locales/de.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"Show a card grid of posts": "Zeige Artikel als Karten an",
-	"Show a feed list of posts": "Zeige Artikel als Feed an",
+	" Show a card grid of posts": " Zeige Artikel als Karten an",
```
Comment (tq-bit) [inline content/themes/candidus/locales/de.json:2]: A lot of i18n values have an extra space, please remove these

File: content/themes/candidus/locales/en.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-	"Show a card grid of posts": "Show a card grid of posts",
-	"Show a feed list of posts": "Show a feed list of posts",
+	" Show a card grid of posts": " Show a card grid of posts",
```
Comment (tq-bit) [inline content/themes/candidus/locales/en.json:2]: A lot of i18n values have an extra space, please remove these

**Review by tq-bit — CHANGES_REQUESTED** [review]
Hey there. Thank you so much for your contribution and the translation :) 

In the en & de file, some extra spaces emerged. Perhaps due to your IDE configuration. Please resolve these and I'll merge your PR

**Comment by matribeiro15** [comment]
I added the spaces because they are present in some .hbs files.
An example is in the share.hbs file.
Since all the sentences there started with a space and not in the json file, the translation helper was not working.

**Comment by matribeiro15** [comment]
this is an example.

**Comment by tq-bit** [comment]
Ahh, alright, thanks for the heads up. I'll open a separate issue about this one. 
Thanks again for your help Mat

**Pull request merged**